WordPress/Classes/ViewRelated/Post/EditImageDetailsViewController.m

```diff
@@ -147,6 +151,17 @@ - (NSArray *)sizeTitles
     return _sizeTitles;
 }
 
+- (CGSize)sizeForSize:(CGSize)size constrainedToRatio:(CGFloat)ratio
```

Comment:
@aerych I think this method is not quite working... portrait images are getting squished in wp-admin (landscape seem to be fine)